doc/howto/network_acls.md

```diff
@@ -92,22 +92,155 @@ lxc query --request GET /1.0/network-acls/my-acl
 
 ## Create an ACL
 
-Use the following command to create an ACL:
+Network ACL names must meet the following requirements:
+
+- Must be between 1 and 63 characters long.
+- Can contain only ASCII letters (a–z, A–Z), numbers (0–9), and dashes (-).
+- Cannot begin with a digit or a dash.
+- Cannot end with a dash.
+
+`````{tabs}
+````{group-tab} CLI
+
+To create an ACL, run:
+
+```bash
+lxc network acl create <ACL_name> [user.KEY=value ...] 
+```
+
+- You must provide an ACL name that meets the requirements described above.
+- You can optionally provide one or more custom `user` keys to store metadata or other information.
+
+ACLs have no rules upon creation via command line, so as a next step, {ref}`add rules <network-acls-rules>` to the ACL. You can also {ref}`edit the ACL configuration <network-acls-edit>`, or {ref}`assign the ACL to a network or NIC <network-acls-assign>`.
+
+Another way to create ACLs from the command line is to provide a YAML configuration file:
+
+```bash
+lxc network acl create <ACL_NAME> < <filename.yaml>
+```
+
+This file must contain at least the ACL `name`, and it can include any other {ref}`network-acls-properties`, including the `egress` and `ingress` properties for defining {ref}`ACL rules <network-acls-rules>`. See the final example in the set below.
+
+### Examples
+
+Create an ACL with the name `acl1`:
+
+```bash
+lxc network acl create acl1
```

Comment:
in the earlier example for showing an ACL you use `my-acl` but here you're using `acl1`, it'd be nice to use consistent example names.